app/src/pages/SellingPage/BookInformation.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Book } from 'cremona/dist/Book';
+import { Box } from '@mui/material';
+import InformationTextLine from '../../components/InformationTextLine';
+
+interface BookInformationProps {
+  book: Book | undefined;
+  name?: string;
+  year?: string;
+  isbn?: string;
+  authors?: string[];
+}
+
+/**
+ * This component is used in the CheckInformationCard that shows all relevant information of a book
+ * @param book : Book \ this books information will be shown if not overriden by the other parameters
+ * @param name : string? \ Overrides the name of the book if defined
+ * @param year : string? \ Overrides the year of the book if defined
+ * @param isbn : string? \ Overrides the isbn of the book if defined
+ * @param authors : string[ ]? \ Overrides the authors of the book if defined
+ *
+ * @returns BookInformation component
+ */
+export default function BookInformation({
+  book,
+  name = undefined,
```

Comment:
Doesn't "?" after the keys in the "BookInformationProps" already make them default to undefined if they are not set by the component using it.